corehq/apps/hqmedia/views.py

```diff
@@ -186,13 +186,57 @@ def upload_controllers(self):
 
     def get(self, request, *args, **kwargs):
         if request.GET.get('json', None):
-            return JsonResponse({
-                "references": self.app.get_references(),
-                "object_map": self.app.get_object_map(),
-                "totals": self.app.get_reference_totals(),
-            })
+            only_missing = bool(int(request.GET.get('only_missing', 0)))
+            include_total = bool(int(request.GET.get('include_total', 0)))
```

Comment:
I don't know the correct/standard pattern for this, but it took some thinking for me to understand that the default is `False` for these two parameters. 

This also breaks hard if someone passes a non numeric value. is the pythonic thing to do just rely on the truthy/falsey-ness of values?

Reply:
I also don't know what the best practice is and don't especially like any of the options. Updated in 5145914362c too at least not fail hard if something non-numeric is passed.

Reply:
Yeah, I started to suggest something, and just got sad with the potential options